packages/main/src/MultiInput.js

```diff
@@ -318,6 +327,20 @@ class MultiInput extends Input {
 		}, 0);
 	}
 
+	onBeforeRendering(...params) {
+		super.onBeforeRendering(...params);
+
+		if (this.showValueHelpIcon) {
+			this.style.setProperty(
+				"--_ui5-input-icons-count",
+				(Number(this.style.getPropertyValue("--_ui5-input-icons-count")) || 0) + 1,
+			);
+		}
+
+		this.tokenizerAvailable = this.tokens && this.tokens.length > 0;
+		this.style.setProperty("--_ui5_input_has_tokens", this.tokenizerAvailable ? 1 : 0);
```

Comment:
Why are you setting additional CSS var if it could only be 1 or 0?

Is there a reason why you can't just write the following CSS: 
```
:host {
	min-width: calc(var(--_ui5_input_min_width) + (var(--_ui5-input-icons-count)*var(--_ui5_input_icon_width)) 
}

:host[tokenizer-available] {
	min-width: calc(var(--_ui5_input_min_width) + (var(--_ui5-input-icons-count)*var(--_ui5_input_icon_width)) + var(--_ui5_input_tokenizer_min_width)));
}

Instead of the current:
:host {
	min-width: calc(var(--_ui5_input_min_width) + (var(--_ui5-input-icons-count)*var(--_ui5_input_icon_width)) + (var(--_ui5_input_has_tokens)*var(--_ui5_input_tokenizer_min_width)));
}
```